control/autoware_autonomous_emergency_braking/src/node.cpp

```diff
@@ -467,7 +467,9 @@ bool AEB::checkCollision(MarkerArray & debug_markers)
       use_pointcloud_data_ && points_belonging_to_cluster_hulls &&
       !points_belonging_to_cluster_hulls->empty()) {
       const auto current_time = obstacle_ros_pointcloud_ptr_->header.stamp;
-      getClosestObjectsOnPath(path, current_time, points_belonging_to_cluster_hulls, objects);
+      const bool ego_moves_forward = current_v > 0.0;
```

Comment:
@danielsanchezaran 
How about using `isDrivingForward` util function to judge driving forward or not?
If trajectory is not subscribed, then check velocity directly is ok. The concern is velocity estimation's chattering.